getting_started.md

```diff
@@ -93,24 +93,23 @@ We can update the configuration with identifiers from the the runtime's **device
 
 Acquire supports up to two video streams.
 These streams consist of a **source** (i.e., a camera), optionally a **filter**, and a **sink** (an output, like a Zarr dataset or a Tiff file).
-Streams are configured independently, so when you get a copy of the runtime configuration, e.g.,
+Before configuring the streams, grab the current configuration of the `Runtime` object with:
 
 ```python
 config = runtime.get_configuration()
 ```
 
-Configure the first video stream by setting properties on `config.video[0]` and the second video stream with `config.video[1]`. We'll be using simulated cameras, one generating a radial sine pattern and one generating a random pattern.
-
+Video streams are configured independently. Configure the first video stream by setting properties on `config.video[0]` and the second video stream with `config.video[1]`. We'll be using simulated cameras, one generating a radial sine pattern and one generating a random pattern.
 
 ```python
-config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sin")
+config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sine")
```

Comment:
```suggestion
config.video[0].camera.identifier = dm.select(acquire.DeviceKind.Camera, "simulated: radial sin")
```
This is actually how it's specified, e.g., [here](https://github.com/acquire-project/acquire-driver-common/blob/main/src/basics.driver.c#L87).